docs/edge.rst

```diff
@@ -318,19 +318,177 @@ the form.
 
 .. image:: _assets/img/cloud-edge-delete.png
    :alt: CrateDB Edge deletion confirmation screen
+    :scale: 20%
```

Comment:
This bit does not work, but is unimportant - @amotl and I will work on this later. You can ignore it

Reply:
Hi Matthijs,

thank you for tagging me on this conversation. This is a classic off-by-one error. When aligning the indentation of `:scale:` to match the first property `:alt:`, i.e. by removing a single space character, it starts working on my machine.

```suggestion
   :scale: 60%
```

With kind regards,
Andreas.

Reply:
Well there you go :D Thanks!